meta/src/parser.rs

```diff
@@ -1171,7 +1171,7 @@ mod tests {
             parser: PestParser,
             input: "0",
             rule: Rule::grammar_rules,
-            positives: vec![Rule::grammar_rule, Rule::grammar_doc],
+            positives: vec![Rule::EOI, Rule::grammar_rule, Rule::grammar_doc],
```

Comment:
Because the `grammar_rules` now is not requirement at least one grammar rule, so this test need to change.